homeassistant/components/tuya/__init__.py

```diff
@@ -50,17 +54,35 @@
 )
 
 
-def setup(hass, config):
+def setup(hass, config, retry_delay=FIRST_RETRY_TIME):
     """Set up Tuya Component."""
 
+    _LOGGER.info("Initializing %s domain.", DOMAIN)
```

Comment:
Please remove this log. It's already logged by the core.

Reply:
I put this log because the core log only the first time, was to have the log also for the retry temptative. Sure I have to remove?

Reply:
Ok, but lower to debug level and don't specify the domain again.